charts/security-apps/README.md

```diff
@@ -55,9 +55,9 @@ This chart is maintained by [Adfinis](https://adfinis.com/?pk_campaign=github&pk
 | gangway.chart | string | `"gangway"` | Chart |
 | gangway.destination.namespace | string | `"infra-gangway"` | Namespace |
 | gangway.enabled | bool | `false` | Enable gangway |
-| gangway.repoURL | string | [repo](https://charts.helm.sh/stable) | Repo URL |
-| gangway.targetRevision | string | `"0.4.*"` | [gangway Helm chart](https://github.com/helm/charts/tree/master/stable/gangway/) version |
-| gangway.values | object | [upstream values](https://github.com/helm/charts/tree/master/stable/gangway/values.yaml) | Helm values |
+| gangway.repoURL | string | [repo](https://gabibbo97.github.io/charts) | Repo URL |
```

Comment:
i got a 4o4 on https://gabibbo97.github.io/charts/ is that a normal behaviour?

Reply:
not normal, but it has not got a index.html the https://gabibbo97.github.io/charts/index.yaml is there to

Reply:
alright, that's look good to me then :)

Reply:
yes normal, ours is fancy af ;)